modules/wso2greg/templates/5.2.0/bin/ciphertool.sh.erb

```diff
@@ -0,0 +1,134 @@
+#!/bin/sh
```

Comment:
Doesn't 5.2.0 ship a `ciphertool.sh` file? I don't see a need for this template if it does.